### PR TITLE
guider: add a new runtime profiler package

### PR DESCRIPTION
**Pull request by iipeace**
guider [1] is a system-wide performance analyzer for platforms based
on linux kernel. it can measure vary system resource usage and
trace thread/process activities. and it reports vary meaningful
output to analyze system status.

[1] https://github.com/iipeace/guider

Signed-off-by: Peace Lee <iipeace5@gmail.com>

### CODE REVIEW

**Comment by iipeace** [comment]
@kraj, could you check new patches?

File: meta-oe/recipes-devtools/guider/guider.bb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+SUMMARY = "runtime performance analyzer"
+HOMEPAGE = "https://github.com/iipeace/guider"
+BUGTRACKER = "https://github.com/iipeace/guider/issues"
+AUTHOR = "Peace Lee <ipeace5@gmail.com>"
+
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
+
+SRC_URI = "git://github.com/iipeace/${BPN}"
+SRCREV = "${AUTOREV}"
```
[the remark this answers is not in the document]
Comment (iipeace) [reply meta-oe/recipes-devtools/guider/guider.bb]: okay, I'll fix it :)

File: meta-oe/recipes-devtools/guider/guider.bb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+GUIDER_SCRIPT = "guider"
+
+do_install() {
+    python ${S}/setup.py install
+
+    install -d ${D}${bindir}
+    install -v -m 0755 ${R}/${bindir}/${GUIDER_SCRIPT} ${D}${bindir}/${GUIDER_SCRIPT}
+
+    install -d ${D}${datadir}/${BPN}
+    install -v -m 0755 ${R}/${datadir}/${BPN}/${GUIDER_OBJ} ${D}${datadir}/${BPN}/${GUIDER_OBJ}
```
[the remark this answers is not in the document]
Comment (iipeace) [reply meta-oe/recipes-devtools/guider/guider.bb]: setup.py will install some files to into python site-packages directory.
Is there any environment variable to define python site-packages directory?

File: meta-oe/recipes-devtools/guider/guider.bb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+inherit distutils
+
+GUIDER_OBJ = "guider.pyc"
+GUIDER_SCRIPT = "guider"
+
+do_install() {
+    python ${S}/setup.py install
+
+    install -d ${D}${bindir}
+    install -v -m 0755 ${R}/${bindir}/${GUIDER_SCRIPT} ${D}${bindir}/${GUIDER_SCRIPT}
```
[the remark this answers is not in the document]
Comment (iipeace) [reply meta-oe/recipes-devtools/guider/guider.bb]: setup.py will install some files to into python site-packages directory.
Is there any environment variable to define python site-packages directory?

**Comment by iipeace** [comment]
@kraj,
I pushed a new commit that reflects your opinion :)
could you check it again?

**Comment by kraj** [comment]
applied 1828df678ccb9a019281446f9bb17b632e8ad636

**Pull request closed without merging**